collectors/apps.plugin/apps_plugin.c

```diff
@@ -3304,6 +3320,14 @@ void send_resource_usage_to_netdata(usec_t dt) {
             , (unsigned int)(cminflt_fix_ratio * 100 * RATES_DETAIL)
             , (unsigned int)(cmajflt_fix_ratio * 100 * RATES_DETAIL)
             );
+    
+    fprintf(stdout,
+        "BEGIN apps.files %llu\n"
+        "VARIABLE fdperc = %llu\n"
+        "END\n"
+        , dt
+        , usedfdpercentage
+        );
```

Comment:
A [better place](https://github.com/netdata/netdata/blob/master/collectors/apps.plugin/apps_plugin.c#L3605-L3609) to send the value of the variable.